morpheus/_lib/cmake/libraries/cuda_utils.cmake

```diff
@@ -18,15 +18,16 @@ find_package(pybind11 REQUIRED)
 
 # Place the two cuda sources in their own target and disable IWYU for that target.
 add_library(cuda_utils_objs
-OBJECT
-      ${MORPHEUS_LIB_ROOT}/src/utilities/matx_util.cu
-      ${MORPHEUS_LIB_ROOT}/src/utilities/type_util.cu
+  OBJECT
+    ${MORPHEUS_LIB_ROOT}/src/utilities/matx_util.cu
+    ${MORPHEUS_LIB_ROOT}/src/utilities/type_util.cu
 )
 
-
 set_target_properties(
-  cuda_utils_objs
+    cuda_utils_objs
     PROPERTIES
+      CUDA_STANDARD 17
+      CUDA_STANDARD_REQUIRED ON
```

Comment:
Why is this necessary?

Reply:
Because nvcc doesn't support c++20 for the versions we use. If we don't set this it inherits from CMAKE_CXX_STANDARD, and CMake fails because it doesn't know how to set CUDA20 compiler parameters for nvcc.